src/concepts/metric_hub.md

```diff
@@ -316,6 +347,32 @@ Statistics on top of these metrics need to be defined in the [`looker/` folder i
 
 To get more statistics added, please reach out on the [#data-help](https://mozilla.slack.com/archives/C4D5ZA91B) Slack channel.
 
+To filter and segment metrics in Looker, data sources that expose fields as dimensions can be configured in metric-hub. These base field data sources need to be joined with the metric data sources. Wildcard characters can be used to apply these joins to multiple data sources:
+
+```toml
+[data_sources.looker_base_fields]
+select_expression = """
+    SELECT
+        submission_date,
+        client_id,
+        os,
+        country,
+        channel
+    FROM
+        mozdata.telemetry.clients_daily
+"""
+columns_as_dimensions = true  # expose the selected fields as dimensions in Looker
+
+# Join `looker_base_fields` on to all the data sources in the file
+# The selected fields in `looker_base_fields` will show up as dimensions for all the metrics
+[data_sources.'*'.joins.looker_base_fields]
+
+# Overwrite the join, to allow for a different data source to be used as base field data source
```

Comment:
It seems to me that this would not overwrite the join `data_sources.baseline.joins.looker_base_fields`, but instead add another join in addition to the one that was defined above via the wildcard join to `looker_base_fields` (`data_sources.baseline.joins.looker_base_fields`). Based on the comment, I'm guessing my understanding is off?

Reply:
This will actually overwrite the `joins` for the `baseline` data source to only join on `some_other_datasource`. I figured overwriting makes sense when working with wildcards, otherwise it's only possible to add more joins, but not actually remove/replace them

Reply:
Hm, yea that makes sense in terms of functionality, even if it's perhaps a bit confusing to remember that sometimes it's an addition and sometimes an overwrite, depending on whether wildcards are involved. I agree with your decision though, so we probably just want to make it extra clear in the docs.